routes/login/login.controller.js

```diff
@@ -13,6 +13,10 @@ module.exports = function(app) {
   //SIN
   app.get('/login/sin', (req, res) => res.render('login/sin', { data: req.session || {} }))
   app.post('/login/sin', validateRedirect, checkSchema(sinSchema), postSIN)
+
+  //Date of Birth
+  app.get('/login/dateOfBirth', (req, res) => res.render('login/dob', { data: req.session || {} }))
+  app.post('/login/dateOfBirth', validateRedirect, checkSchema(birthSchema), postDoB)
```

Comment:
I think we should either

- change the URL to `/dob` to match the template
- change the template to `dateOfBirth.pug` to match the URL